lib/common/common/src/validation.rs

```diff
@@ -180,6 +180,23 @@ pub fn validate_multi_vector<T>(multivec: &[Vec<T>]) -> Result<(), ValidationErr
     Ok(())
 }
 
+pub fn validate_multi_vector_len(
+    vector_count: u32,
+    flatten_dense_vector: &[f32],
+) -> Result<(), ValidationErrors> {
+    let dense_vector_len = flatten_dense_vector.len();
+    if dense_vector_len % vector_count as usize != 0 {
```

Comment:
Validate please `vector_count > 0`. User may set count as zero in grpc `Vector` type